tools/cldr-apps/js/src/views/LoginButton.vue

```diff
@@ -35,7 +50,34 @@ export default {
         // now reload this page now that we've logged out
         await window.location.reload();
       } else {
-        window.location.replace("./login.jsp");
+        if (this.userName && this.password) {
+          try {
```

Comment:
subroutine would be better than inserting so much code into existing function

Reply:
yes, the login stuff should go under esm modules